src/NfOrd/Unit/FindUnits.jl

```diff
@@ -259,6 +259,7 @@ function _unit_group_find_units(u::UnitGrpCtx, x::ClassGrpCtx; add_orbit::Bool =
   end
   @vprint :UnitGroup 1 "Finished processing\n"
   if has_full_rank(u)
+    u.units = reduce(u.units, u.tors_prec)
```

Comment:
Eh, this is unrelated to the other changes?

Reply:
Oh my bad. I forgot to reverse it.